base/applications/regedit/lang/pt-PT.rc

```diff
@@ -103,35 +103,35 @@ BEGIN
         BEGIN
             MENUITEM "&Chave", ID_EDIT_NEW_KEY
             MENUITEM SEPARATOR
-            MENUITEM "Valor da &cadeia", ID_EDIT_NEW_STRINGVALUE
-            MENUITEM "Valor &Binário", ID_EDIT_NEW_BINARYVALUE
+            MENUITEM "&Valor da cadeia", ID_EDIT_NEW_STRINGVALUE
+            MENUITEM "Valor &binário", ID_EDIT_NEW_BINARYVALUE
             MENUITEM "Valor &DWORD", ID_EDIT_NEW_DWORDVALUE
             MENUITEM "Valor &Multi-cadeia", ID_EDIT_NEW_MULTISTRINGVALUE
-            MENUITEM "Valor c&adeia expansível", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
+            MENUITEM "Valor de cadeia e&xpansível", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
         END
         MENUITEM SEPARATOR
         MENUITEM "&Permissões...", ID_EDIT_PERMISSIONS
         MENUITEM SEPARATOR
-        MENUITEM "&Excluir\tDel", ID_EDIT_DELETE
-        MENUITEM "&Mudar o nome", ID_EDIT_RENAME
+        MENUITEM "Elimina&r\tDel", ID_EDIT_DELETE
+        MENUITEM "M&udar o nome", ID_EDIT_RENAME
         MENUITEM SEPARATOR
         MENUITEM "&Copiar nome de chave", ID_EDIT_COPYKEYNAME
         MENUITEM SEPARATOR
-        MENUITEM "&Localizar...\tCtrl+F", ID_EDIT_FIND
-        MENUITEM "L&ocalizar seguinte\tF3", ID_EDIT_FINDNEXT
+        MENUITEM "&Localizar...\tCtrl+L", ID_EDIT_FIND
```

Comment:
I would assume that just changing the Ctrl+F to Ctrl+L here will not actually make the regedit really listen to that accelerator. So I would recommend to undo that change.
```suggestion
        MENUITEM "&Localizar...\tCtrl+F", ID_EDIT_FIND
```